content/docs/2.5/concepts/scaling-jobs.md

```diff
@@ -48,6 +48,7 @@ spec:
       - "Ready"
       - "PodScheduled"
       - "AnyOtherCustomPodCondition"
+    MultipleScalersOption: "max" # Optional. Default: max. Specifies how to calculate the queueLength and maxValue when multiple scaler are used.
```

Comment:
```suggestion
    multipleScalersOption: "max" # Optional. Default: max. Specifies how to calculate the queueLength and maxValue when multiple scaler are used.
```

Reply:
What about naming this property `multipleScalersCalculation` or something like that?

Reply:
Sure. I'll change the code.